### DISCUSSION TITLE
build error with babel-plugin-module-resolver

### DISCUSSION THREAD

**Discussion by emilforsmann** [vercel/vercel#4289] [General]
Hi guys,

Is anybody else experiencing issue with using babel-plugin-module-resolver?

getting this error: `Module not found: Can't resolve '../../../layout/main' in '/zeit/7b754faf/pages/[id]/settings'`

It's a Nextjs project and running `build` locally or running `now dev` works like a charm. 
I've also tried removing package-lock.json and node_module and doing a clean build, the problem still remains. 

This is the specific module on pages/[id]/settings:
`import Main from 'layout/Main';`
and this is how module-resolver is specified:
```json
{
    "root": ["./"],
    "alias": {
     "components": "./components",
     "contexts": "./contexts",
     "hooks": "./hooks",
     "hoc": "./hoc",
     "lib": "./lib",
     "layout": "./layout",
     "mutations": "./mutations",
     "queries": "./queries"
    }
  }
```

**Comment by amytych** [comment]
FWIW, if your folders are in the root you don't need to explicitly set aliases, once you set `"root": ["./"],` you should be able to import from the root folders, e. g. this should work `import Main from 'layout/Main';`

It might be that the explicit aliases are causing issues.
